worlds/test/issue_3003_distortion_implementation_correction.world

```diff
@@ -0,0 +1,65 @@
+<sdf version='1.7'>
+  <world name='default' xmlns:ignition='http://ignitionrobotics.org/schema'>
+    <scene>
+      <shadows>0</shadows>
+      <ambient>0.4 0.4 0.4 1</ambient>
+      <background>1 1 1 1</background>
+    </scene>
+    <model name='minecart'>
+      <static>1</static>
+      <link name='base_link'>
+        <collision name='collision'>
+          <geometry>
+            <mesh>
+              <uri>/root/.ignition/fuel/fuel.ignitionrobotics.org/openrobotics/models/mine cart/4/meshes/minecart.dae</uri>
```

Comment:
these paths with `/root` are not valid on my machine. I recommend the following instead:

~~~
https://fuel.ignitionrobotics.org/1.0/openrobotics/models/mine cart/4/files/meshes/minecart.dae
~~~

Reply:
Thanks for pointing this out.